config.go

```diff
@@ -536,13 +530,22 @@ func LoadConfig(path string) (*Config, error) {
 			}
 		case "outputs":
 			for outputName, outputVal := range subtbl.Fields {
-				outputSubtbl, ok := outputVal.(*ast.Table)
-				if !ok {
-					return nil, err
-				}
-				err = c.parseOutput(outputName, outputSubtbl)
-				if err != nil {
-					return nil, err
+				switch outputSubtbl := outputVal.(type) {
```

Comment:
Small nit, `outputSubtbl` feels awkward (same with `tbl` in the rest of this function, but that's out of scope). `outputSubTbl` or even `outputSubTable` would be more readable.